backend/src/exception_views.py

```diff
@@ -0,0 +1,21 @@
+from flask import Blueprint, request, jsonify
+from werkzeug import exceptions
+
+
+exception = Blueprint("exception", __name__)
+
+
+@exception.app_errorhandler(exceptions.InternalServerError)
+def _handle_internal_server_error(ex):
+    if request.path.startswith("/api/"):
```

Comment:
what's the justification for this statement? shouldnt everything be prefixed with this due to the middleware?

Reply:
Yes you are correct. Nice catch.